agent-cypress/package.json

```diff
@@ -2,13 +2,14 @@
   "name": "@visual-regression-tracker/cypress-examples",
   "version": "1.0.0",
   "description": "",
+  "type": "module",
   "scripts": {
     "test": "cypress run"
   },
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@visual-regression-tracker/agent-cypress": "^5.1.0",
-    "cypress": "^9"
+    "@visual-regression-tracker/agent-cypress": "*",
```

Comment:
please, upgrade to `5.7.0`
https://github.com/Visual-Regression-Tracker/agent-cypress/releases/tag/v5.7.0